src/test/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2Test.java

```diff
@@ -385,6 +390,8 @@ void shouldRequireReauthentication_ifNonceIsExpired() throws JSONException {
             final HttpServletResponse response = Mockito.spy(new MockHttpServletResponse());
             final JSONObject tokenResponse = new JSONObject();
             tokenResponse.put("id_token", idToken);
+            tokenResponse.put("tokenHint", "blah");
```

Comment:
does this expect a tokenHint? I thought the tokenHint was just the idToken. Same below as well

Reply:
Yeah these are not actually needed - have removed